cmd/katapult/main_test.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/spf13/cobra"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func testAssertCommand(t *testing.T, cmd *cobra.Command, errResult, want, stderrResult string) {
```

Comment:
I know I suggested the name `testAssertCommand`, but the more I'm reading it now, it makes me think it's testing an "assertion command" 🤦‍♂️... So what do you think about calling it something like `assertCobraCommand` to make it more specific about it's purpose and function?

Along the lines of making things clearer, what do you think about renaming the three string arguments to `wantErr`, `wantStdout`, and `wantStderr`?

And finally, I think we should call `t.Helper()` at the beginning of the function to denote it as a helper function.